apps/react-ts-host/@mf-types/react-ts-remote/_types/apps/react-ts-remote/src/remote-entry.d.ts

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Is this supposed to be html?

Reply:
Thanks for catching this, not sure how I overlooked it. Part of this PR was trying to fix:

@nrwl/react:module-federation-dev-server

Where this just redirects using the history fallback API, for now I removed the types but I need to take a look how to fix this or look at their source code and override it